libc/test/include/CMakeLists.txt

```diff
@@ -483,3 +483,69 @@ add_libc_test(
   DEPENDS
     libc.include.llvm-libc-macros.math_function_macros
 )
+
+# Test `#include <...>` of each header in each available language mode.
+# This is gated on -DLLVM_LIBC_BUILD_HEADER_TESTS=ON until all the bugs
+# in headers are fixed so the tests all compile.
+set(TEST_STDC_VERSIONS 89;99;11;17;23)
```

Comment:
Should this be C90?
```suggestion
set(TEST_STDC_VERSIONS 90;99;11;17;23)
```

Reply:
For each of the versions there are multiple spellings that are documented to mean the same thing.  c89 and c90 are two of the spellings for the same thing (along with others), but c89 is the one used most commonly.
(Similar for c17/c18.)